apps/editing-toolkit/editing-toolkit-plugin/starter-page-templates/page-template-modal/index.js

```diff
@@ -231,11 +231,6 @@ class PageTemplateModal extends Component {
 		}
 
 		this.setTemplate( name );
-
-		// Turn off sidebar's instance of modal
-		if ( this.props.isPromptedFromSidebar ) {
-			this.props.toggleTemplateModal();
```

Comment:
I've glad the call to this method is gone! I can't find a reference to it anywhere, looks like it would have crashed!

Reply:
😱